#!/bin/sh -e
## 11_fix-misc-gcc-warnings.dpatch by Henrique de Moraes Holschuh <hmh@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix bad C code that causes warnings and strict aliasing rules problems

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch}"

case "$1" in
       -patch) patch $patch_opts -p1 < $0;;
       -unpatch) patch $patch_opts -p1 -R < $0;;
        *)
                echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
                exit 1;;
esac

exit 0

@DPATCH@
diff -urNad hplip/pcard/pcardext/pcardext.c /tmp/dpep.dYipcz/hplip/pcard/pcardext/pcardext.c
--- hplip/pcard/pcardext/pcardext.c	2005-05-13 14:36:42.750140133 -0300
+++ /tmp/dpep.dYipcz/hplip/pcard/pcardext/pcardext.c	2005-05-19 13:58:11.331592479 -0300
@@ -20,7 +20,7 @@
 Requires:
 Python 2.2+
 
-/*****************************************************************************/
+\*****************************************************************************/
 
 #include <Python.h>
 #include <structmember.h>
